### ISSUE TITLE
Support transfer of entire folders

### ISSUE THREAD

**Issue by augustogoncalves** [Autodesk-Forge/bim360appstore-data.management-nodejs-transfer.storage#1]
As of July/17, the creation of folders is not supported on BIM 360 Docs. This feature is planned.

**Comment by augustogoncalves** [comment]
supporting Autodesk>Storage transfer of folders

**Comment by augustogoncalves** [comment]
implemented Storage>Autodesk

**Comment by kudzuman** [comment]
You say it is implemented but it is not working for me from DropBox storage to Autodesk. Also, you have not updated the Instructions in Help to demonstrate how to move folders that may be empty. I need this tool to set up standard folder structures from a template folder structure in DropBox.

**Comment by augustogoncalves** [comment]
@kudzuman, correct, it's not transferring empty folders, only if at least one file is available. Will reopen this issue for further investigation.
